docs/wfss.rst

```diff
@@ -0,0 +1,112 @@
+.. _wfss:
+.. sectionauthor:: Russell Ryan <rryan@stsci.edu>
+.. codeauthor:: Russell Ryan <rryan@stsci.edu>
+
+Wide-Field Slitless Spectroscopy
+================================
+
+:term:`wide-field slitless spectroscopy` (WFSS) refers to either the act of obtaining a complete, unbiased spectroscopic view of the sky by means of tramissive and dispersive optical element (usually a :term:`grism` or :term:`prism`) **and** the data produced by such an observation.  This is to distinguish other ways of using these optical elements, such as spatial/drift scanning, spectro-polarimetry, or detailed transient spectroscopy, which are not implemented in ``slitlessutils``.  It is unclear if/when such modes will be implemented.
+
+Below the two main *container* datatypes that one generally interacts with are the :class:`~slitlessutils.wfss.WFSS()` and :class:`~slitlessutils.wfss.WFSSCollection()` to hold a single WFSS file/image and a set of such objects.  Since some of the extraction techniques implemented in ``slitlessutils`` can be computationally expensive and/or use a significant amount of memory.  Therefore, care has been taken to limit the memory footprint of objects, for example the image-objects do not instantiate the data until requested.
```

Comment:
I think you're right that something is amiss.  But instead of taking your suggestion as is, I added the word "respectively".  I think that is maybe better.  Cool?